TurnkeySecurity/IotDeviceGuardBitLocker.psm1

```diff
@@ -25,15 +25,16 @@ function GenerateBitlockerDataRecoveryAgentPackageXml([xml] $config)
         $blob = (Get-ItemProperty HKLM:\SOFTWARE\Microsoft\SystemCertificates\My\Certificates\$thumbprint).Blob
 
         # convert 'blob' from array to 'int' to a comma delimited string of hex values
```

Comment:
is this comment appropriate anymore?  says a comma delimited string, but the join isn't using a comma anymore?

Reply:
Updated